content/news/2024-10-09-ESG-WP5-NOBUGS2024/index.md

```diff
@@ -8,9 +8,12 @@ subsites: [all-eu, esg]
 main_subsite: eu
 ---
 
+
+<img class="center" src="./NOBUGS_2024.png" width="80%" alt="NOBUGS Welcome page">
```

Comment:
You can add images in markdown and add an alt text if you like.

Reply:
You mean by  ``` ![Alt text](the url path)``` . Shall I change it or the html is ok?

Reply:
Yes, please use markdown whenever you can.